common/PacketTypes.asn1

```diff
@@ -0,0 +1,29 @@
+PacketTypes DEFINITIONS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    PUSC-UINT, PUSC-BOOL FROM BasicTypes
+    ApplicationProcess-ID FROM ApplicationProcess;
+
+
+ApplicationProcess-ID-Packet ::= ApplicationProcess-ID
+SecondaryHeaderFlag ::= PUSC-BOOL
+
+Packet-ID {PacketType} ::= SEQUENCE
+{
+    packetType PacketType,
+    secondaryHeaderFlag SecondaryHeaderFlag,
+    applicationProcess-ID ApplicationProcess-ID-Packet
+}
+
+TM-Packet-ID ::= Packet-ID {NULL}
+TC-Packet-ID ::= Packet-ID {NULL}
```

Comment:
this looks little strange, but I get how it happened ;)

maybe it could be made a little bit more readable with some additional type

Reply:
it would also be nice to have separate files for TC and TM related things, so user would not need to import both of them